python/Ganga/Utility/feedback_report.py

```diff
@@ -652,6 +653,15 @@ def renameDataFiles(directory):
                 logger.debug("Err %s" % str(err))
                 writeErrorLog(str(sys.exc_info()[1]))
 
+        # Copy thread stack trace file
+        try:
+            thread_trace_source_path = os.path.join(getConfig('Configuration')['gangadir'], thread_trace_file_name)
+            thread_trace_target_path = os.path.join(fullLogDirName, thread_trace_file_name)
+            shutil.copyfile(thread_trace_source_path, thread_trace_target_path)
```

Comment:
I did it this way so that it reuses the existing `thread_trace.html` file that's on disk rather than creating a new one. I'm not averse to changing this to create a new file though if that's what people prefer?